tests/test_extract_metricvalues.py

```diff
@@ -127,14 +127,14 @@ def check_stdout(kind, n, last_line, contained, **kwargs):
         # default call with no kwargs
         contained = "{instrument: 'TestCam', detector: 25, visit: 54321, ...}"
         last = "verify.testing: 42.0"
-        check_stdout("value", 9, last, contained)
+        check_stdout("value", 11, last, contained)
```

Comment:
I think I understand adding one output (because of the extra metric added in 688b131), but why does this one increase by two?

Reply:
Two lines get printed: one for the dataId, the other for the metricvalues.